dependency-check-maven-config.xml

```diff
@@ -286,6 +286,7 @@
             gwc-web-1.5.0.war: commons-collections-3.1.jar
             gwc-web-1.5.0.war: postgresql-8.4-701.jdbc3.jar
             geowebcache-server-standalone-0.7.0.war: gwc-sqlite-1.9.1.jar
+            geowebcache-server-standalone-0.7.0.war: sqlite-jdbc-3.8.6.jar
```

Comment:
It might just be because of the way that alliance builds on top of DDF, so I would expect to need to exclude it on alliance as well. However, these should be popping up on DDF. If they aren't, then it sounds like our OWASP plugin isn't working quite right over there.